docs/src/main/asciidoc/build-analytics.adoc

```diff
@@ -0,0 +1,186 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Build analytics
+:categories: analytics
+:summary: This guide presents what build analytics is and how to configure it.
+
+The Quarkus team has limited knowledge, from Maven download numbers, of the remarkable growth of Quarkus and the number of users reporting issues/concerns. Still, we need more insight into the platforms, operating system, java combinations, and build tools our users employ.
+The build analytics tool aims to provide us with this information.
+
+== How it works
+
+*Collection happens only at build-time* and not at runtime. Collection occurs during usage of Quarkus build tooling (Maven, Gradle). Your built application does not contain any data collection code.
+
+*All data collected is anonymous*. There is no way to link this data back to an individual or organization. The intent is to learn about general patterns and trends; therefore, only anonymous data is needed.
+
+*Collection is Disabled by default* You will be asked if you want to opt in to data collection the first time you use devMode. The console will wait for a few seconds for your input.
+
+== This is the data being collected
+
+* Anonymous application name. SHA-256 hash of the Group Id + Artifact Id.
+* Anonymous application version. SHA-256 hash of the Artifact Id + Version.
+* Anonymous user id. During the 1st build, this is generated locally at random, and subsequently reused.
+* Quarkus version
+* Java version
+* GraalVM version
+* Operating system details
```

Comment:
I think you should list what we collect here because this can be sensitive.

Reply:
Will do

Reply:
On a follow up PR